nagios/README.md

```diff
@@ -9,36 +9,86 @@ service_perfdata_file_template=DATATYPE::SERVICEPERFDATA\tTIMET::$TIMET$\tHOSTNA
+                                                           --service \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_api_token [CSP_API_TOKEN] \
+                                                           --csp_base_url [CSP_BASE_URL]
+}
+```
+
+#### Using CSP Oauth
+```
+define command {
+    command_name  wavefront-perf-host
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]//host-perfdata \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_app_id [CSP_APP_ID] \
+                                                           --csp_app_secret [CSP_APP_SECRET] \
+                                                           --csp_base_url [CSP_BASE_URL]                                                        
+}
+
+define command {
+    command_name  wavefront-perf-service
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
+                                                           --service \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_app_id [CSP_APP_ID] \
+                                                           --csp_app_secret [CSP_APP_SECRET] \
+                                                           --csp_base_url [CSP_BASE_URL]
+}
+```
+
+#### Using Wavefront Ingestion
+```
+define command {
+    command_name  wavefront-perf-host
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
                                                            --wf_proxy_addr [WAVEFRONT_PROXY_ADR] \
                                                            --wf_proxy_port [WAVEFRONT_PROXY_PORT]
-}
+ }
+
 
 define command {
-    command_name  wavefront_perf_service
-    command_line  /opt/nagios/etc/wf/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
-                                                          --service \
-                                                          --wf_server [WAVEFRONT_URL] \
-                                                          --wf_token [WAVEFRONT_TOKEN]
+    command_name  wavefront-perf-service
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
+                                                           --service \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --wf_token [WAVEFRONT_TOKEN]
 }
 ```
 
-### Events
+Refer the configuration samples at https://github.com/wavefrontHQ/integrations/raw/master/nagios/config to send metrics using CSP API token, CSP Oauth or Wavefront API Token.
```

Comment:
...using a VMware Cloud services API token, server to server OAuth app credentials, or an Operations for Applications API token.